cinema/tests/test_movie_view_set.py

```diff
@@ -0,0 +1,135 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from rest_framework import status
+from rest_framework.reverse import reverse
+from rest_framework.test import APIClient
+
+from cinema.models import Movie, Genre, Actor
+from cinema.serializers import MovieListSerializer, MovieDetailSerializer
+
+MOVIE_URL = reverse("cinema:movie-list")
+
+
+def test_movie(**params) -> Movie:
+    movie = {
+        "title": "Test Movie",
+        "description": "Test",
+        "duration": 120,
+    }
+    movie.update(params)
+    return Movie.objects.create(**movie)
+
+
+def test_genre(**params) -> Genre:
+    genre = {"name": "Drama"}
+    genre.update(params)
+    return Genre.objects.create(**genre)
+
+
+def test_actor(**params) -> Actor:
+    actor = {"first_name": "Tom", "last_name": "Hardy"}
+    actor.update(params)
+    return Actor.objects.create(**actor)
```

Comment:
Consider renaming this function to `create_test_actor` to avoid confusion with test case methods.